File: 01_Mon/prac.py
def prac_loops():
    num = int(input("Pick a number: "))
    for number in range(num):
        if number != 0:
            if number % 2 == 0:
                print(f" {number} : Even")

            else:
                print(f" {number} : Odd")
        else:
            print("Sorry... no ")


def prac_diff_loops():
    num = int(input("Pick a number: "))
    i = 1
    while (i < num):
        if i != 0:
            if i % 2 == 0:
                print(f" {i} : Even")

            else:
                print(f" {i} : Odd")
        i += 1


def main():
    prac_loops()
    prac_diff_loops()


if __name__ == "__main__":
    main()

File: 01_Mon/prac.js
function pracLoops() {
    var baseNum = [1, 2, 3, 4, 5, 6, 7, 8];
    baseNum.forEach(function (element) {
        if (element != 0) {
            if (element % 2 == 0) {
                console.log(element + ": Even")
            } else {
                console.log(element + ": Odd")
            }
        }
    })
}

function pracDiffLoop() {
    num = 24
    baseNum = 1;

    while (baseNum < num) {
        if (baseNum != 0) {
            if (baseNum % 2 == 0) {
                console.log(`${baseNum} : Even`)
            } else {
                console.log(`${baseNum} : Odd`)
            }
        }
        baseNum += 1;
    }



}

pracLoops()
pracDiffLoop()